core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/FromClause.kt

```diff
@@ -2,18 +2,33 @@ package ch.ergon.dope.resolvable.clause.model
 
 import ch.ergon.dope.DopeQuery
 import ch.ergon.dope.DopeQueryManager
+import ch.ergon.dope.resolvable.Resolvable
 import ch.ergon.dope.resolvable.clause.ISelectClause
 import ch.ergon.dope.resolvable.clause.ISelectUnnestClause
 import ch.ergon.dope.resolvable.formatToQueryStringWithSymbol
+import ch.ergon.dope.resolvable.fromable.AliasedBucket
 import ch.ergon.dope.resolvable.fromable.Fromable
 
-class FromClause(private val fromable: Fromable, private val parentClause: ISelectClause) : ISelectUnnestClause {
+class FromClause : ISelectUnnestClause {
+    private val resolvable: Resolvable
```

Comment:
why resolvable? An AliasedBucket is a fromable as well

Reply:
The `ReferencedAliasedBucket` isn't.

Reply:
Can't we move the call ._addBucketReference()_ inside the _toDopeQuery_? then we would get rid of these additional constructors

Reply:
done